.github/workflows/main.yaml

```diff
@@ -42,17 +42,19 @@ jobs:
         run: npm install
         
       - name: tsc
-        run: npm run build
+        run: npm run build -w packages
 
-      - name: Create and iniytialize Postgres database
+      - name: Create and iniitialize Postgres database
         # Runs a script that creates a PostgreSQL table, populates
         # the table with data, and then retrieves the data.
-        run: node build/tools/database.js create && node build/tools/database.js init      
+        run: node packages/dbadmin/dist/tools/database.js create && node packages/dbadmin/dist/tools/database.js init      
       
       - name: start server and run tests (one step, else server will be down when tests start)
         run: |
+          cd packages/core
```

Comment:
I think we do not need to step into the subdirectory, but we can specify in the top-level package.json the dev and test tasks as `npm run -w @lionweb/repository-core dev` and `npm run -w @lionweb/repository-dbadmin test`

Reply:
Agreed, that is much more clear.